examples/chat/enterprise/aws-bedrock-anthropic/app.py

```diff
@@ -4,18 +4,17 @@
 # To get started, follow the instructions at https://aws.amazon.com/bedrock/claude/
 # as well as https://github.com/anthropics/anthropic-sdk-python#aws-bedrock
 # ------------------------------------------------------------------------------------
-from pathlib import Path
-
 from anthropic import AnthropicBedrock
-from dotenv import load_dotenv
 
 from shiny.express import ui
 
+# In Shiny Core, do `from app_utils import load_dotenv`
+from .app_utils import load_dotenv
```

Comment:
I think this should work as an absolute import, right? At least after the changes in #1464.

```suggestion
from app_utils import load_dotenv
```

Reply:
Yea, I was going back and forth on this. It does work as an absolute import, but pyright/pylance complains about it. Thinking about it a bit more, I think I'd prefer this?

```python
from app_utils import load_dotenv  # pyright: ignore[reportMissingImports]
```